File: Haskell/beginner/1157.hs
-- https://judge.beecrowd.com/pt/problems/view/1153
-- Fatorial Simples

-- Getting System
import System.IO

-- Main Function
main :: IO ()
main = do

    -- Getting user input
    usr_input <- readLn :: IO Int

    -- Printing results
    mapM_ print [i | i <- [1.. usr_input], usr_input `mod` i == 0]

-- By Fernando Serra
-- https://github.com/fernandosserra

File: Python/beginner/1157.py
# https://judge.beecrowd.com/pt/problems/view/1157
# Divisores I

usr_input = int(input())

for i in range(1, usr_input + 1):
    if usr_input % i == 0:
        print(f'{i}')
        
# By Fernando Serra
# https://github.com/fernandosserra